.gitignore

```diff
@@ -10,6 +10,9 @@ Desktop.ini
 ## Ignore Visual Studio temporary files, build results, and
 ## files generated by popular Visual Studio add-ons.
 
+# Roslyn temporary files
+*.sln.ide/
```

Comment:
Let's use the same rule that's in the official repo: `*.ide/`

https://github.com/github/gitignore/blob/master/VisualStudio.gitignore#L22-L23